### PR TITLE
[pt] Updates the module lesson, with instropection for structs

### PR DESCRIPTION
**Pull request by ogabriel**
Hi.

I was re-reading the module lesson, but this time in portuguese, and I noticed that it was missing the part that talks about instrospection, since it is hacktoberfest, I felt like it is a good time to updated it, so I've done.

I have also bumped the version of the lession from 1.3.0 > 1.4.0, since it is on pair with the english version now.

### CODE REVIEW

File: pt/lessons/basics/modules.md [hunk 1]
```diff
@@ -102,6 +102,35 @@ iex> %{name: "Sean"} = sean
...
+```elixir
+defmodule Example.User do
+  @derive {Inspect, only: [:name]}
+  defstruct name: nil, roles: []
+end
+```
+
+*Nota:* podemos também usar `@derive {Inspect, except: [:roles]}`, que é equivalente.
+
+Com o nosso modulo já atualizado, vamos ver o que acontece no `iex`:
```
Comment (vbrazo) [inline pt/lessons/basics/modules.md]:
```suggestion
Com o nosso módulo já atualizado, vamos ver o que acontece no `iex`:
```

File: pt/lessons/basics/modules.md [hunk 2]
```diff
@@ -102,6 +102,35 @@ iex> %{name: "Sean"} = sean
...
 ```
 
+A partir do Elixir 1.8, structs incluem introspecção customizáveis. Para entender o que isso significa e como devemos usar, vamos inspecionar nossa captura `sean`:
+
+ ```elixir
+ iex> inspect(sean)
+"#Example.User<name: \"Sean\", roles: [...], ...>"
+```
+
+Todos os campos estão presentes, o que esta correto para esse exemplo, mas o que acontece se tivéssemos um campo protegido que não gostaríamos de incluir? A nova funcionalidade `@derive` faz com que possamos alcançar isso! Vamos atualizar nosso exemplo para que `roles` não seja mais incluído na nossa saída:
```
Comment (thiamsantos) [inline pt/lessons/basics/modules.md]:
```suggestion
Todos os campos estão presentes, o que está correto para esse exemplo, mas o que acontece se tivéssemos um campo protegido que não gostaríamos de incluir? A nova funcionalidade `@derive` faz com que possamos alcançar isso! Vamos atualizar nosso exemplo para que `roles` não seja mais incluído na nossa saída:
```

File: pt/lessons/basics/modules.md [hunk 3]
```diff
@@ -102,6 +102,35 @@ iex> %{name: "Sean"} = sean
 #Example.User<name: "Sean", roles: [...], ...>
 ```
 
+A partir do Elixir 1.8, structs incluem introspecção customizáveis. Para entender o que isso significa e como devemos usar, vamos inspecionar nossa captura `sean`:
```
Comment (thiamsantos) [inline pt/lessons/basics/modules.md]: Could you please, update to have each phrase on its own line? Related #1652.

Comment (ogabriel) [reply pt/lessons/basics/modules.md]: This new rule will be applied to all the `pt` files in the future?

Comment (thiamsantos) [reply pt/lessons/basics/modules.md]: Yes, probably @ogabriel. I'm not sure if we need to update everything to follow the guideline, but we can update whenever we update a translation.

**Comment by thiamsantos** [comment]
Thanks @ogabriel !

**Pull request merged**